doc/api/tls.md

```diff
@@ -1031,26 +1116,37 @@ console.log(tls.getCiphers()); // ['AES128-SHA', 'AES256-SHA', ...]
 ## Deprecated APIs
 
 ### Class: CryptoStream
-
+<!-- YAML
+added: v0.3.4
+deprecated: v0.11.3
+-->
     Stability: 0 - Deprecated: Use [`tls.TLSSocket`][] instead.
 
 The `tls.CryptoStream` class represents a stream of encrypted data. This class
 has been deprecated and should no longer be used.
 
 #### cryptoStream.bytesWritten
-
+<!-- YAML
+added: v0.3.4
+deprecated: v0.11.3
+-->
 The `cryptoStream.bytesWritten` property returns the total number of bytes
 written to the underlying socket *including* the bytes required for the
 implementation of the TLS protocol.
 
 ### Class: SecurePair
-
+<!-- YAML
+deprecated: v0.11.3
+-->
     Stability: 0 - Deprecated: Use [`tls.TLSSocket`][] instead.
 
 Returned by `tls.createSecurePair()`.
 
 #### Event: 'secure'
-
+<!-- YAML
+added: v0.3.2
+deprecated: v0.11.3
```

Comment:
Are you sure this one is deprecated? It isn’t tagged as deprecated like e.g. `createSecurePair` is